<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DO_Editer</class>
 <widget class="QDialog" name="DO_Editer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>727</width>
           <height>161</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>21</x>
            <y>20</y>
            <width>24</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>通道</string>
          </property>
         </widget>
         <widget class="QComboBox" name="ChannelCBX">
          <property name="geometry">
           <rect>
            <x>102</x>
            <y>21</y>
            <width>69</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>21</x>
            <y>60</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>平台ID</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="SubPrjNameID">
          <property name="geometry">
           <rect>
            <x>101</x>
            <y>60</y>
            <width>133</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="SubPrjName">
          <property name="geometry">
           <rect>
            <x>101</x>
            <y>90</y>
            <width>133</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>21</x>
            <y>90</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>点位名</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="SubPrjNum">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>120</y>
            <width>133</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>120</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>分项编号</string>
          </property>
         </widget>
         <widget class="QLabel" name="Linklable">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>90</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>获取平台参数</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>725</width>
           <height>186</height>
          </rect>
         </property>
         <property name="title">
          <string>报警开关</string>
         </property>
         <widget class="QCheckBox" name="EnableCBX">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>ENABLE</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>页</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>711</width>
           <height>81</height>
          </rect>
         </property>
         <property name="title">
          <string>报警</string>
         </property>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>44</y>
            <width>598</width>
            <height>18</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="DI_DO_OUT_Layout">
           <item>
            <widget class="QCheckBox" name="DI_DO1_OUT">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO2_OUT">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO3_OUT">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO4_OUT">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO5_OUT">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO6_OUT">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO7_OUT">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO8_OUT">
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO9_OUT">
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO10_OUT">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO11_OUT">
             <property name="text">
              <string>11</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO12_OUT">
             <property name="text">
              <string>12</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO13_OUT">
             <property name="text">
              <string>13</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO14_OUT">
             <property name="text">
              <string>14</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO15_OUT">
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO16_OUT">
             <property name="text">
              <string>16</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_3">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>20</y>
            <width>598</width>
            <height>18</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="DI_DO_ENB_Layout">
           <item>
            <widget class="QCheckBox" name="DI_DO1_ENB">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO2_ENB">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO3_ENB">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO4_ENB">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO5_ENB">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO6_ENB">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO7_ENB">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO8_ENB">
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO9_ENB">
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO10_ENB">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO11_ENB">
             <property name="text">
              <string>11</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO12_ENB">
             <property name="text">
              <string>12</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO13_ENB">
             <property name="text">
              <string>13</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO14_ENB">
             <property name="text">
              <string>14</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO15_ENB">
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="DI_DO16_ENB">
             <property name="text">
              <string>16</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>54</width>
            <height>12</height>
           </rect>
          </property>
          <property name="text">
           <string>高限输出</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>54</width>
            <height>12</height>
           </rect>
          </property>
          <property name="text">
           <string>高限使能</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ConfirmBtn">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelBtn">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  
  <zorder>tabWidget</zorder>
  <zorder>CancelBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
